arbitrum-docs/node-running/troubleshooting-running-nodes.md

```diff
@@ -1,9 +1,159 @@
+                <p>The guidance displayed on this page will change based on your selected configuration.</p>
+            </div>
+        </div>
+        <div className='task'>
+            <div className='input-container'><input id="tc-3" type='checkbox'/><span className='done'></span></div>
+            <div className='guidance-container'>
+                <label htmlFor="tc-3">3. Review the FAQ</label>
+                <p>Answers to frequently asked questions can be found in <a href="/node-running/faq">FAQ: Run a node</a>.</p>
+            </div>
+        </div>
+        <div className='task'>
+            <div className='input-container'><input id="st-1" type='checkbox'/><span className='done'></span></div>
+            <div className='guidance-container'>
+                <label htmlFor="st-1">4. TODO - this changes with the OS</label>
+                <Tabs className="tabgroup-with-label" groupId="os" defaultValue="others" values={[
+                    {label: 'Operating system:', value: 'label'},
+                    {label: 'Linux, MacOS, Arm64', value: 'others'},
+                    {label: 'Windows', value: 'win'}
+                    ]}>
+                    <TabItem className="unclickable-element" value="label"></TabItem>
+                    <TabItem value="others"><p>Demo -  others</p></TabItem>
+                    <TabItem value="win"><p>Demo -  windows</p></TabItem>
+                </Tabs>
+            </div>
+        </div>
+        <div className='task'>
+            <div className='input-container'><input id="st-2" type='checkbox'/><span className='done'></span></div>
+            <div className='guidance-container'>
+                <label htmlFor="st-2">5. TODO - this changes with the network</label>
+                <Tabs className="tabgroup-with-label network-tabgroup" groupId="network" defaultvalue="arb-one" values={[   {label: 'Network:', value: 'label'},
+                            {label: 'Localhost', value: 'localhost'},
+                            {label: 'Arbitrum Goerli', value: 'arb-goerli'},
+                            {label: 'Arbitrum One', value: 'arb-one'},
+                            {label: 'Arbitrum Nova', value: 'arb-nova'}
+                        ]}>
+                    <TabItem className="unclickable-element" value="label"></TabItem>
+                    <TabItem value="localhost"><p>Demo -  localhost</p></TabItem>
+                    <TabItem value="arb-goerli"><p>Demo -  goerli</p></TabItem>
+                    <TabItem value="arb-one"><p>Demo -  one</p></TabItem>
+                    <TabItem value="arb-nova"><p>Demo -  nova</p></TabItem>
+                </Tabs>
+            </div>
+        </div>
+    </div>
+</div>
 
-<div data-faq-origin-slug='node-faq'>
-    <FaqPartial />
 </div>
 
-export const toc = FAQTOC
+
+### Step 2: Look for your scenario
+
+Common troubleshooting scenarios and solutions are detailed below.
+
+<table class='small-table'>
```

Comment:
Pulled in from the partial; we can add it back to the partial if/when we need to display it in more than one place (eg in a node-type-specific doc)